src/tree.jl

```diff
@@ -160,8 +160,7 @@ end
 Register a Dantzig-Wolfe decomposition on the JuMP model `model` where the index-set
 of the subproblems are `axis`.
```

Comment:
This isn't in the changes, but the tutorial added in "axis.jl" made me think if it would make a difference to put an index-set like L in the `axis` parameter instead of the actual axis K. If there's a difference between these two options, maybe "where the index-set of the subproblems is in `axis`" or something similar would help with the ambiguity?

Reply:
If you use `L` it won't work. The argument should be an axis.